src/event-stream/event-server.ts

```diff
@@ -41,6 +46,10 @@ async function handleMempoolTxsMessage(rawTxs: string[], db: DataStore): Promise
     };
   });
   for (const tx of decodedTxs) {
+    // Send update to websocket subscribers
+    tx_subscribers
```

Comment:
Client websocket related code shouldn't be necessary in this file. Tx status updates should be sent to clients after the db layer has completed processing. The `datastore.addListener('txUpdate', ...)` hook emits mempool updates, so this can just be removed.